greenboot-rs.spec

```diff
@@ -8,6 +8,7 @@ Summary:	Generic Health Check Framework for systemd
 License:	(Apache-2.0 OR MIT) AND BSD-3-Clause
 URL:		https://github.com/fedora-iot/greenboot-rs
 Source0:	%{url}/releases/download/%{version}/%{name}-%{version}.tar.gz
+Source1:            %{name}-%{version}-vendor-patched.tar.xz
```

Comment:
can we wrap this around `%if 0%{?centos}` too?

Reply:
Yeah, it's considered a bad practice to conditionalize the sources: https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_do_not_conditionalize_sources